airbyte-integrations/connectors/source-chargebee/unit_tests/integration/test_addon.py

```diff
@@ -195,7 +195,7 @@ def test_given_no_initial_state_when_read_then_return_state_based_on_most_recent
         output = self._read(_config().with_start_date(self._start_date - timedelta(hours=8)), _NO_STATE)
         most_recent_state = output.most_recent_state
         assert most_recent_state.stream_descriptor == StreamDescriptor(name=_STREAM_NAME)
-        assert most_recent_state.stream_state == AirbyteStateBlob(updated_at=cursor_value)
+        assert most_recent_state.stream_state == AirbyteStateBlob(updated_at=str(cursor_value))
```

Comment:
The state has changed from a int to a string. This shouldn't be an issue because as shown [here](https://github.com/airbytehq/airbyte/pull/48510/files#diff-c50715739b0c617eafa1a113334f6f981c6f89a6672dea31dffddfa2cca6c922R205), the change is backward compatible